### PR TITLE
[FEATURE] Auto tap uband recommendation

### PR DESCRIPTION
**Pull request by nitbharambe**

### CODE REVIEW

File: docs/user_manual/components.md [hunk 1]
```diff
@@ -753,7 +753,7 @@ The actual grid state is not changed after calculations are done.
 The following additional requirements exist on the input parameters.
 
 - The automatic tap changer algorithm currently only supports tap changers connected at HV side of the transformer. Hence, the rated voltage of the node at the `tap_side` must be greater than or equal to the rated voltage of the node at the other side of transformer.
-- The voltage band must be sufficiently large
+- The voltage band must be sufficiently large. It would depend on the resultant voltage being transformed. But generally when the resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```
Comment (mgovers) [inline docs/user_manual/components.md:756]:
```suggestion
- The voltage band must be sufficiently large. Although it depends on the resultant voltage being transformed, if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```

File: docs/user_manual/components.md [hunk 2]
```diff
@@ -753,7 +753,7 @@ The actual grid state is not changed after calculations are done.
 The following additional requirements exist on the input parameters.
 
 - The automatic tap changer algorithm currently only supports tap changers connected at HV side of the transformer. Hence, the rated voltage of the node at the `tap_side` must be greater than or equal to the rated voltage of the node at the other side of transformer.
-- The voltage band must be sufficiently large
+- The voltage band must be sufficiently large. Although it depends on the resultant voltage being transformed, if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```
Comment (Jerry-Jinfeng-Guo) [inline docs/user_manual/components.md:756]:
```suggestion
- Depending on the resultant voltage being transformed, the voltage band must be sufficiently large: if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```

**Review by mgovers — APPROVED** [review]

**Comment by nitbharambe** [comment]
Also check https://github.com/PowerGridModel/power-grid-model/issues/594 and https://github.com/PowerGridModel/power-grid-model/issues/595#issuecomment-2114982172

File: docs/user_manual/components.md [hunk 3]
```diff
@@ -753,7 +753,7 @@ The actual grid state is not changed after calculations are done.
 The following additional requirements exist on the input parameters.
 
 - The automatic tap changer algorithm currently only supports tap changers connected at HV side of the transformer. Hence, the rated voltage of the node at the `tap_side` must be greater than or equal to the rated voltage of the node at the other side of transformer.
-- The voltage band must be sufficiently large
+- Depending on the resultant voltage being transformed, the voltage band must be sufficiently large: if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```
Comment (nitbharambe) [inline docs/user_manual/components.md:756]:
```suggestion
- Depending on the resultant voltage being transformed, the voltage band must be sufficiently large: At zero line drop compensation if the expected resultant voltages are in the proximity of the rated transformer voltages, it is recommended to have the $u_{band} >= \frac{u_2}{1+ u_1 / \text{tap}_{\text{size}}}$
```

**Review by mgovers — APPROVED** [review]

**Pull request merged**